README.md

```diff
@@ -182,18 +182,17 @@ and toolchains that are currently known to work (last updated 2022/01/24):
 ### Prerequisites
 
 - [Boost](https://www.boost.org/): program_options
-- [ROOT](https://root.cern/): RIO, Hist, Tree
 
 ## Getting started
 
 ### Clone the repository
 
-Clone the repository and setup up the submodules, this requires `git-lfs` for the data from the `traccc-data` repository.
+Clone the repository and setup up the data directory.
```

Comment:
_setup up_? I notice this was here before, but let's fix it while we have the chance.